samples/android/GleanCrashExample/app/src/main/java/org/mozilla/gleancrashexample/MainActivity.kt

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.gleancrashexample
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import android.view.Menu
+import android.view.MenuItem
+
+import kotlinx.android.synthetic.main.activity_main.fab
+import kotlinx.android.synthetic.main.activity_main.toolbar
+import mozilla.components.service.glean.Glean
+import org.mozilla.gleancrashexample.GleanMetrics.Crash
+import org.mozilla.gleancrashexample.GleanMetrics.Pings
+import java.lang.NullPointerException
+
+class MainActivity : AppCompatActivity(), Thread.UncaughtExceptionHandler {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+        setSupportActionBar(toolbar)
+
+        fab.setOnClickListener {
+            // Go boom!
+            throw NullPointerException()
+        }
+
+        Glean.setUploadEnabled(true)
+        Glean.initialize(this)
+        Glean.registerPings(Pings)
+
+        Thread.setDefaultUncaughtExceptionHandler(this)
+    }
+
+    override fun uncaughtException(p0: Thread, p1: Throwable) {
```

Comment:
nit: what's `p0` and `p1`? :) Let's give them clearer names, e.g. `terminatedThread`, `occurredException`